view-creator-framework/src/test/java/org/hypertrace/core/viewcreator/pinot/PinotUtilsTest.java

```diff
@@ -59,54 +60,57 @@ public void testCreatePinotTableForView() {
         new File(this.getClass().getClassLoader()
             .getResource("sample-view-generation-spec.conf").getPath())));
 
-    final TableConfig tableConfig = PinotUtils.createPinotTable(viewCreationSpec);
+    final TableConfig tableConfig = PinotUtils.createPinotTableConfig(viewCreationSpec);
 
     LOGGER.info("Pinot Table Config for View: {}", tableConfig);
-    Assertions.assertEquals(tableConfig.getTableName(), "myView1_REALTIME");
-    Assertions.assertEquals(tableConfig.getTableType(), TableType.REALTIME);
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getLoadMode(), "MMAP");
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getStreamConfigs().get("streamType"),
-        "kafka");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.consumer.type"),
-        "LowLevel");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.topic.name"),
-        "test-view-events");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.consumer.factory.class.name"),
-        "org.apache.pinot.plugin.stream.kafka20.KafkaConsumerFactory");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.decoder.class.name"),
-        "org.apache.pinot.plugin.inputformat.avro.confluent.KafkaConfluentSchemaRegistryAvroMessageDecoder");
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getStreamConfigs()
-        .get("stream.kafka.hlc.zk.connect.string"), "localhost:2181");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.zk.broker.url"),
-        "localhost:2181");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.broker.list"),
-        "localhost:9092");
-    Assertions.assertEquals(
-        tableConfig.getIndexingConfig().getStreamConfigs().get("stream.kafka.decoder.prop.schema.registry.rest.url"),
-        "http://localhost:8081");
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getStreamConfigs()
-        .get("realtime.segment.flush.threshold.time"), "3600000");
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getStreamConfigs()
-        .get("realtime.segment.flush.threshold.size"), "500000");
-    Assertions.assertEquals(tableConfig.getIndexingConfig().getStreamConfigs()
-        .get("stream.kafka.consumer.prop.auto.offset.reset"), "largest");
+    Assertions.assertEquals("myView1_REALTIME", tableConfig.getTableName());
```

Comment:
The order of `expected` and `actual` arguments to `assertEquals` method was wrong here so I have fixed them all to be correct.

Reply:
I thought I had fixed when I was moving this around.

Reply:
yes, other places are good.